sopel/plugins/capabilities.py

```diff
@@ -194,6 +194,8 @@ def register(self, plugin_name: str, request: capability) -> None:
         """Register a capability ``request`` for ``plugin_name``.
 
         :param request: the capability request to register for later
+        :raise RuntimeError: when the capability request is too long for a
+                             single ``CAP REQ`` and ``CAP * ACK``.
```

Comment:
```suggestion
                             single ``CAP REQ`` and ``CAP * ACK``
```

Style nit: existing `:raise:` declarations in other files have no trailing period